cmd/config-manager/main.go

```diff
@@ -456,8 +456,8 @@ func findPidToSignal(f *Flags) (int, error) {
 	return -1, fmt.Errorf("no process found")
 }
 
-func fileExists(filename string) (bool, error) {
-	info, err := os.Stat(filename)
+func symlinkExists(filename string) (bool, error) {
+	info, err := os.Lstat(filename)
```

Comment:
Although this change makes sense, is the `return !info.IsDir(), nil` at the end of the function make sense?

Reply:
I think it does, as in the `updateSymlink` function we want to point to a config file, not to a directory. 

Maybe the name of the function is a bit misleading and could be something like `symlinkFileExists`, what do you think?

Reply:
I suppose my quesion should then be, why not check the symlink mode in `info`:
```
	info, err := os.Lstat(filename)
	if err != nil {
		return filename, fmt.Errorf("failed to get file info: %v", info)
	}
	if info.Mode()&os.ModeSymlink == 0 {
		return filename, fmt.Errorf("not a symlink: %v", filename)
	}
```

Reply:
If we just check the mode it means that, when`filename` is a path to a symlink pointing to a directory, this function returns `true`, right? 

If so, is this the intended behavior? Updating the config-manager providing a symlink that points to a directory would make it break, so it makes sense to check whether the provided symlink actually points to a file. Maybe I'm missing something?